DOCKER.md

```diff
@@ -0,0 +1,87 @@
+## Docker
+
+In order to successfully test Qt.py against the different bindings of different versions, we use Travis-CI to run Docker containers based on [pre-built CentOS-based images](https://hub.docker.com/r/fredrikaverpil/qt.py/tags/).
+
+The Docker images follow the [VFX Reference Platform specifications](http://www.vfxplatform.com/) with some additionals, and are quite jam-packed.
+
+
+<br>
+<br>
+
+
+**Software built from source**
+
+* glibc<sup>1</sup>
+* gcc<sup>1</sup>
+* git<sup>2</sup>
+* cmake<sup>3</sup>
+* Qt4<sup>4</sup>
+* Qt5 + Autodesk-modifications<sup>1</sup>
+* Qt5 Creator<sup>1, 5</sup>
+* SIP<sup>4</sup>
+* PyQt4<sup>4</sup>
+* PySide<sup>4</sup>
+* PyQt5<sup>1</sup>
+* PySide2<sup>1</sup>
+
+<sup>1</sup> Per specification from VFX Platform  
+<sup>2</sup> Adds possibility to faster clone large repositories  
+<sup>3</sup> cmake 3.x required to build PySide2  
+<sup>4</sup> Required for Qt.py testing  
+<sup>5</sup> Required for `PySide2.QtUiTools`
+
+<br>
+<br>
+
+
+**Software versions**
+
+We use source code from around the time of [SIGGRAPH](http://www.siggraph.org) (usually July/August) each year. This is usually when the VFX Reference Platform is updated.
```

Comment:
This is fine for now, but we should be more specific in the future. Do they not have dates associated with each release? Or a commit made somewhere of the release?